examples/basic.py

```diff
@@ -45,6 +30,26 @@ def on_event(event: EdgeEvent, event_data: MariNode | Frame):
 )
 def main(port: str | None, log_dir: str):
     """A basic example of using the MariLib library."""
+    logger_container = {"instance": None}
+
+    def on_event(event: EdgeEvent, event_data: MariNode | Frame):
+        logger = logger_container["instance"]
+        if not (logger and logger.active and event_data):
+            return
+
+        if event == EdgeEvent.NODE_JOINED:
+            # print(f"Node {event_data} joined")
+            # print("#", end="", flush=True)
+            logger.log_event(event_data.address, "NODE_JOINED")
+        elif event == EdgeEvent.NODE_LEFT:
+            # print(f"Node {event_data} left")
+            # print("0", end="", flush=True)
+            logger.log_event(event_data.address, "NODE_LEFT")
```

Comment:
Instead of using the string `"NODE_LEFT"`, I think it's possible to directly convert the enum `EdgeEvent.NODE_LEFT` to a string.